traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -0,0 +1,340 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/lib/go-tc"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/auth"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/tovalidate"
+	"github.com/go-ozzo/ozzo-validation"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+)
+
+// TODeliveryServiceRequest provides a type alias to define functions on
+type TODeliveryServiceServer tc.DeliveryServiceServer
+
+//the refType is passed into the handlers where a copy of its type is used to decode the json.
+var refType = TODeliveryServiceServer(tc.DeliveryServiceServer{})
+
+func GetRefType() *TODeliveryServiceServer {
+	return &refType
+}
+
+func (dss TODeliveryServiceServer) GetKeyFieldsInfo() []api.KeyFieldInfo {
+	return []api.KeyFieldInfo{{"deliveryservice", api.GetIntKey}, {"server", api.GetIntKey}}
+}
+
+//Implementation of the Identifier, Validator interface functions
+func (dss TODeliveryServiceServer) GetKeys() (map[string]interface{}, bool) {
+	if dss.DeliveryService == nil {
+		return map[string]interface{}{"deliveryservice": 0}, false
+	}
+	if dss.Server == nil {
+		return map[string]interface{}{"server": 0}, false
+	}
+	keys := make(map[string]interface{})
+	ds_id := *dss.DeliveryService
+	server_id := *dss.Server
+
+	keys["deliveryservice"] = ds_id
+	keys["server"] = server_id
+	return keys, true
+}
+
+func (dss *TODeliveryServiceServer) GetAuditName() string {
+	if dss.DeliveryService != nil {
+		return strconv.Itoa(*dss.DeliveryService) + "-" + strconv.Itoa(*dss.Server)
+	}
+	return "unknown"
+}
+
+func (dss *TODeliveryServiceServer) GetType() string {
+	return "deliveryserviceServers"
+}
+
+func (dss *TODeliveryServiceServer) SetKeys(keys map[string]interface{}) {
+	ds_id, _ := keys["deliveryservice"].(int) //this utilizes the non panicking type assertion, if the thrown away ok variable is false i will be the zero of the type, 0 here.
+	dss.DeliveryService = &ds_id
+
+	server_id, _ := keys["server"].(int) //this utilizes the non panicking type assertion, if the thrown away ok variable is false i will be the zero of the type, 0 here.
+	dss.Server = &server_id
+}
+
+// Validate fulfills the api.Validator interface
+func (dss *TODeliveryServiceServer) Validate(db *sqlx.DB) []error {
+
+	errs := validation.Errors{
+		"deliveryservice": validation.Validate(dss.DeliveryService, validation.Required),
+		"server":          validation.Validate(dss.Server, validation.Required),
+	}
+
+	return tovalidate.ToErrors(errs)
+}
+
+//The TODeliveryServiceServer implementation of the Creator interface
```

Comment:
The first word of a function comment should begin with the name of the function, `Create`. Not doing so breaks Godoc.